ui/app/templates/components/auth-form.hbs

```diff
@@ -1,9 +1,4 @@
 <div class="auth-form">
-  {{#if showLoading}}
```

Comment:
Removed per Design's request. We show a loader on the submit button and grey it out.